partner_statement/wizard/activity_statement_wizard.py

```diff
@@ -47,13 +47,14 @@ def _print_report(self, report_type):
             report_name = "p_s.report_activity_statement_xlsx"
         else:
             report_name = "partner_statement.activity_statement"
+        partners = self.env["res.partner"].browse(self._context["active_ids"])
```

Comment:
@MiquelRForgeFlow Don't forget that _context is deprecated :smiley: 

https://github.com/odoo/odoo/blob/14.0/odoo/models.py#L5030

Reply:
Well, when these backward-compatibility things get removed, then the code will be adapted haha